charts/simple-app/templates/poddisruptionbudget.yaml

```diff
@@ -1,3 +1,6 @@
+{{- if lt (.Values.replicaCount | int) 2 }}
+{{- fail "Deployment replica count can not be less than 2 in order to configure PDB. Please configure Replica count greater than 2." }}
+{{- end }}
```

Comment:
This needs to go on the **inside** of the PDB check.

`{{- if .Values.podDisruptionBudget -}}`

Reply:
It works but it should be executed only if PDB configuration is set. Good catch.

Reply:
Should be greater than or equal to 2... not just greater than 2 (in your failure string)

Reply:
updated it.